test_conformance/math_brute_force/CMakeLists.txt

```diff
@@ -15,12 +17,16 @@ set(${MODULE_NAME}_SOURCES
     function_list.h
     i_unary_double.cpp
     i_unary_float.cpp
+    i_unary_half.cpp
     macro_binary_double.cpp
     macro_binary_float.cpp
+    macro_binary_half.cpp
     macro_unary_double.cpp
     macro_unary_float.cpp
+    macro_unary_half.cpp
     mad_double.cpp
     mad_float.cpp
+    mad_half.cpp
```

Comment:
we've added `mad` but we've not added a `ternary_half.cpp` file to test `fma()`.

Similarly we need a `unary_two_resuts_half` to test `sincos`, and `unary_two_results_i_float.cpp` to test `lgamma_r` and `frexp`.

Are half `fma()`, `lgamma_r`, `frexp` being testing in some other way, or are they just missing?

Reply:
I've added missing source files, couple of remarks:

1) Whole test should be unified due to huge duplication of code. I think current progress is not a good starting point for modernization so my proposition is to create new issue.
2) I had to modify the logic adapted from ternary_float in order to pass the test with cl_half. Source of the confusion is Ulp_Half_Error. I've added [additional conditions](https://github.com/KhronosGroup/OpenCL-CTS/blob/04033bf8bd958c9c053e753363c4588f7dc75ccf/test_conformance/math_brute_force/ternary_half.cpp#L293-L294) to work this problem around but I am not sure if that is correct approach.